### PR TITLE
Add Devnet Deploy Script

### PR DESCRIPTION
**Pull request by DaSichuan**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+SOLANA_CONFIG=$1;
+PROGRAM_ID=$2;
+OWNER=$3;
+echo "Using Solana config filepath: $SOLANA_CONFIG"
+echo "Program ID: $PROGRAM_ID"
+echo "OWNER: $OWNER"
+
+solana config set --url https://api.devnet.solana.com
+
+SOURCE=`target/debug/spl-token --config $SOLANA_CONFIG wrap 21 2>&1 | awk '{print $NF}'`
```
Comment (nope-finance) [inline deploy.sh:12]: lets have end of script call unwrap

Comment (DaSichuan) [reply deploy.sh:12]: What do you mean unwrap?

Comment (nope-finance) [reply deploy.sh:12]: also since this is really just for devnet lets only wrap 10 and ask for a 10 airdrop lol?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+solana program --config $SOLANA_CONFIG deploy \
+  --program-id $PROGRAM_ID \
+  target/deploy/spl_token_lending.so
+
+CREATE_MARKET_OUTPUT=`target/debug/spl-token-lending create-market \
+  --fee-payer    $OWNER \
+  --market-owner $OWNER \
```
Comment (nope-finance) [inline deploy.sh:20]: discussed offline changing this back to just the id

File: deploy.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  --fee-payer         $OWNER \
+  --market-owner      $OWNER \
+  --source-owner      $OWNER \
+  --market            $MARKET_ADDR \
+  --source            $SRM_TOKEN_ACCOUNT \
+  --amount            5000000  \
+  --pyth-product      6MEwdxe4g1NeAF9u6KDG14anJpFsVEa2cvr5H6iriFZ8 \
+  --pyth-price        992moaMQKs32GKZ9dxi8keyM2bUmbrwBZpK4p2K6X5Vs \
+  --switchboard-feed  BAoygKcKN7wk8yKzLD6sxzUQUqLvhBV1rjMA4UJqfZuH \
+  --verbose
```
Comment (nope-finance) [inline deploy.sh:74]: nit new line

**Comment by DaSichuan** [comment]
Adds a devnet deploy script. We are messing a lot with the byte format of accounts so this will make it faster and less manual to iterate. 

Invoke with something like `./deploy.sh ./owner_config.yml ./lending.json`

**Pull request merged**